README.md

```diff
@@ -68,9 +66,37 @@ Note: If you enter the Room Name, then you can restrict this user's access to th
 
 <img width="562px" src="images/quickstart/room-connected.png"/>
 
+### Running the Objective-C Quickstart
```

Comment:
QuickStart

Reply:
The other quickstart repos are using `Quickstart`. Leaving as is for the time being unless we want to change it across all quickstart repos.